rabbit_messaging.gemspec

```diff
@@ -22,7 +22,7 @@ Gem::Specification.new do |spec|
   spec.add_runtime_dependency "bunny", "~> 2.0"
   spec.add_runtime_dependency "exception_notification"
   spec.add_runtime_dependency "lamian"
-  spec.add_runtime_dependency "rails"
+  spec.add_runtime_dependency "rails", ">= 5.2.2.1", "~> 5.2"
```

Comment:
Why?

Reply:
Bundler audit did find vulnerabilities

Reply:
But the version isn't locked in master, so why doesn't it use the latest version? 
And why should we force `~> 5.2`? What if I want to upgrade to Rails 6?